### PR TITLE
Switch to Scala 2.12.7

### LINKED ISSUE #26: Provide a Scala 2.12 comptabile release
For moving forward with apache/incubator-openwhisk#3952 we need to release a mesos-actor jar compiled with Scala 2.12.

### PR DESCRIPTION
**Pull request by chetanmeh**
Fixes #26

### CODE REVIEW

**Comment by chetanmeh** [comment]
@tysonnorris Gradle does not seem to [provide a simple way to generate cross builds](https://stackoverflow.com/questions/21450500/cross-build-scala-using-gradle)

I was thinking to make a release with version `0.0.8_2.12` from this branch to see how far we can move in OpenWhisk core repo. Later once we have OpenWhisk core moved to 2.12 we can merge this and then do a proper release 0.0.9 and move OpenWhisk core to that version .

Thoughts?

File: build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ repositories {
 }
 
 ext {
-    scalaVersion = "2.11"
+    scalaVersion = "2.12"
```
Comment (tysonnorris) [inline build.gradle:22]: is there a way to parameterize this, so that we don't have branch drift in the meantime while we wait for 2.12 in OW?

**Comment by tysonnorris** [comment]
Let's first see if there is a way to parameterize the scala version; otherwise yes it seems fine to create a release from this branch

File: build.gradle (hunk 1 above)
Comment (chetanmeh) [reply build.gradle:22]: Can possibly be done but may introduce some complexity. As we may not be needing to support multiple Scala version ... we can keep this simple for now

**Comment by tysonnorris** [comment]
Can we close this PR in the meantime?

**Comment by chetanmeh** [comment]
Closing for now. Would reopen once we are ready to merge to master

**Pull request merged**